plugins/factory-plugin/src/factory-plugin.ts

```diff
@@ -19,6 +20,7 @@ export async function start(context: theia.PluginContext) {
         projectsRoot = projectsRootEnvVar;
     }
 
+    await new WorkspaceManager().run();
```

Comment:
i think that run should be run asynchrounously: we don't need to wait for that to be finished to start the `FactoryInitializer` right ?

Reply:
yeah, you are right, no need.